frontend/src/views/DiagnosticTunnel/index.vue

```diff
@@ -257,6 +269,21 @@ export default {
         }
       })
     },
+    previousAction() {
+      if (!this.formIsValid) return
+      this.saveDiagnostic().then(() => {
+        if (this.previousStep) {
```

Comment:
Tel que le template est structuré aujourd'hui, `this.previousStep` sera toujours vrai. Mais je veux découpler cette décision de la fonction elle même dans le cas où par exemple on voudrait revenir au summary